app/controllers/v1/sessions_controller.rb

```diff
@@ -32,9 +32,13 @@ class SessionsController < ApplicationController
     # Collection Action: auth is required for certain types of requests
     # @type is set automatically by the routes in config/routes.rb
     # For more details see SAML::SSOeSettingsService and SAML::URLService
+    # rubocop:disable Metrics/MethodLength
     def new
       type = params[:type]
       client_id = params[:application] || 'vaweb'
+      operation = params[:operation]
```

Comment:
let's go ahead and give this the default of `authorize` here similar to how the `client_id` above does it for `vaweb`. This way you can run `validate_operation_params` regardless of the input